### ISSUE TITLE
Hi! Great stuff!!! from zero to a working boot device in 2'. Thank you!

### ISSUE THREAD

**Issue by f-bone** [artyomsoft/retro-flash-drive#1]
Only one request and not a problem report, can we somehow make the image for 2gb cf cards (max allowed for fat16). This way a LOT of stuff can fit in there!

thank you for your efforts!

**Comment by extrowerk** [comment]
I agree. I never understood why is there no disk-image which one can simply dd on a flash drive, everybody makes the unnecessary dance with mkfs and whatever needed to make the partition active. Thanks for your work!
